src/inspectorscripts.ts

```diff
@@ -16,14 +16,16 @@ export
 
     static py_script: string = `import json
 from sys import getsizeof
-
 from IPython import get_ipython
 from IPython.core.magics.namespace import NamespaceMagics
 
+
 _jupyterlab_variableinspector_nms = NamespaceMagics()
 _jupyterlab_variableinspector_Jupyter = get_ipython()
 _jupyterlab_variableinspector_nms.shell = _jupyterlab_variableinspector_Jupyter.kernel.shell
 
+_jupyterlab_variableinspector_maxrows = 10**6 # None for view all
```

Comment:
I think it would be better if we move this parameter to the typescript code and pass it via the `get_dict_list` function.

Reply:
my question would be how - as a user, could change this option if it is part of typescript?

Reply:
It should be possible to alter the threshold value directly in the inspector view. For now, we can just hard code some value in the typescript part and then add some configuration method later on. Furthermore we should think about whether we want to limit the number of rows or the number of cells of a matrix. For my part, I think it is best if we limit the number of cells since we cannot control the size of the data to transfer from the kernel only by limiting the number of rows.

Reply:
sure thing, I'll make the changes now